appveyor.yml

```diff
@@ -30,7 +30,7 @@ test_script:
   - "%CONDA% install conda=4.3.22 --channel conda-forge"
   - "%CONDA% create --name \"ibis_%PYTHON_VERSION%\" python=%PYTHON_VERSION% --channel conda-forge"
   - "%ACTIVATE% \"ibis_%PYTHON_VERSION%\""
-  - "pip install -e .\"[sqlite, postgres, visualization, pandas]\""
+  - "pip install -e .\"[sqlite, postgres, visualization, pandas, git+https://github.com/googledatalab/pydatalab@v1.1]\""
```

Comment:
you should just be able to list this as: bigquery (pip should then call the appropriate requires)